safe/report/expressions/map_report.py

```diff
@@ -89,25 +89,10 @@ def legend_title_header_element(feature, parent):
     return header.capitalize()
 
 
-description = tr(
-    'If the impact layer has a distance field, it will return the distance to '
-    'the nearest place in metres.')
-examples = {
-    'distance_to_nearest_place()': '1234'
-}
-help_message = generate_expression_help(description, examples)
-
-
-@qgsfunction(
-    args='auto', group=label_group, usesGeometry=False, referencedColumns=[],
-    help_text=help_message.to_html(), helpText=help_message.to_html())
-def distance_to_nearest_place(feature, parent):
-    """If the impact layer has a distance field, it will return the distance to
-    the nearest place in metres.
-
-    e.g. distance_to_nearest_place() -> 1234
-    """
-    _ = feature, parent  # NOQA
+def get_exposure_summary_layer():
```

Comment:
`get_` `set_` is not common in our code base. We usually remove these prefix.
But if you think it's the correct name, no problem.